utils/results/conversion/summaryparser/summaryparser.go

```diff
@@ -258,6 +258,19 @@ func (sc *CmdResultsSummaryConverter) ParseIacs(iacs ...[]*sarif.Run) (err error
 	return results.ForEachJasIssue(results.CollectRuns(iacs...), sc.entitledForJas, sc.getJasHandler(jasutils.IaC, false))
 }
 
+func (sc *CmdResultsSummaryConverter) ParseMalicious(malicious ...[]*sarif.Run) (err error) {
+	if !sc.entitledForJas {
+		return
+	}
+	if sc.currentScan == nil {
+		return results.ErrResetConvertor
+	}
+	if sc.currentScan.Vulnerabilities.MaliciousResults == nil {
+		sc.currentScan.Vulnerabilities.MaliciousResults = &formats.ResultSummary{}
+	}
+	return results.ForEachJasIssue(results.CollectRuns(malicious...), sc.entitledForJas, sc.getJasHandler(jasutils.MaliciousCode, false))
```

Comment:
you should also modify `getJasHandler` to make sure you can parse the issue type